elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -1439,10 +1439,17 @@ elytron.client-ssl-context.ssl-session.peer-certificates.version=The certificate
 # Operations
 elytron.client-ssl-context.ssl-session.invalidate=Invalidate the SSLSession (Note: This does not terminate current connections, only prevents future connections from joining or resuming this session).
 
+elytron.dynamic-client-ssl-context=A Dynamic SSL context for use on the client side of a connection. It allows to choose ssl context based on peer's host and port information
```

Comment:
s/It allows to choose ssl context based on peer's host and port information/It makes it possible to choose the SSL context to use based on the peer's host and port information

Reply:
Updated